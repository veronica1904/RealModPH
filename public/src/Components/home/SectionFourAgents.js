import React from "react";
import { Card } from "@mui/material";
import "../../style/sectionFourAgents.css";

export default function SectionFourAgents() {
  const data = [
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_2_jkje4o.png",
      name: "alexis",
      lastName: "Duque Duarte",
      phone: "3228101281",
    },
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_2_jkje4o.png",
      name: "alexis",
      lastName: "Duque Duarte",
      phone: "3228101281",
    },
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_2_jkje4o.png",
      name: "alexis",
      lastName: "Duque Duarte",
      phone: "3228101281",
    },
  ];

  return (
    <section className="section-container">
      <div class="contenedor">
        <img src="https://res.cloudinary.com/veronicaduque/image/upload/v1653111006/RealMod/Our_Agents_ixzutp.png" />
        <div class="centrado">
            <span>EXperties is here</span>
            <p>Our Exclusive Agents</p>
            
        </div>
      </div>
      <div className="containerAgents">
        {data.map((item) => (
          <Card sx={{ width: 270, minHeight: 275 }}>
            <img src={item.image} className="image-card-agents" />
            <div className="description-card">
              <p>{item.name}</p>
              <span>{item.lastName}</span>
              <div className="container-contact">
                <img src="https://res.cloudinary.com/veronicaduque/image/upload/v1653259701/RealMod/fi-rr-call-outgoing_uivqzu.png" />
                <p> Call: {item.phone}</p>
              </div>
            </div>
          </Card>
        ))}
      </div>
    </section>
  );
}
