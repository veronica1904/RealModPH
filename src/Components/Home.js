import React from 'react'
import "../style/listar.css"
import '../style/banner.css'


const Home = () => {
  return (
    
    <div className="container-fluid divListar">
 
 <div className="container-logo">
      <center><h9 className='our'>PROPIETIE</h9></center>
         <img src='https://res.cloudinary.com/veronicaduque/image/upload/v1653111130/RealMod/Properties_1_jf2hwh.png' alt="logo"/>
         <div className="ATRE">Featured Listings</div>
        </div>
    
        <div className='container-banner'>

<div className='container-searching'>
    <div className='container-filter'>
        <button type='button'>Sell</button>
        <button type='button'>Buy</button>
        <button type='button'>Rent</button>
    </div>


    <div className='container-bar-searching'>

        <input type='text' name='search' placeholder='Enter keyword here ...' />

        <select>
            <option defaultValue>Select Location</option>
            <option>Colombia</option>
            <option>Venezuela</option>
            <option>EE.UU</option>
            <option>CUBA</option>
            <option>CHILE</option>
            <option>HAITI</option>
            <option>TRINIDAD Y TOBAGO</option>
            <option>PERÃ™</option>
        </select>

        <div className='container-btn-search'>

            <button type='submit'>
             
                Search
            </button>
        </div>

    </div>
</div>
</div>
  
     <div className="container-logo">
      <center><h9 className='our'>OUR PROPIETIE</h9></center>
         <img src='https://res.cloudinary.com/veronicaduque/image/upload/v1653110895/RealMod/Properties_jelzvq.png' alt="logo"/>
        
        </div>
      <div className="ATRE">Our Featured Properties</div>
     
    </div >
    

  )
}

export default Home