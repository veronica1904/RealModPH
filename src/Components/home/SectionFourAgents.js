import React from "react";
import { Card } from "@mui/material";
import "../../style/sectionFourAgents.css";

export default function SectionFourAgents() {
  const data = [
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_jecjz7.png",
      name: "Andren Willium",
      lastName: "Sweet  Home",
      phone: "+121 511 4522",
    },
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_1_iubeup.png",
      name: "Polly Martzinger",
      lastName: "Sweet  Home",
      phone: "+121 511 4522",
    },
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254948/RealMod/Rectangle_47_2_jkje4o.png",
      name: "Sarah Boysen",
      lastName: "Marl Street",
      phone: "+121 511 4522",
    },
    {
      image:
        "https://res.cloudinary.com/veronicaduque/image/upload/v1653254947/RealMod/Rectangle_47_3_vod2mj.png",
      name: "Mac. Willium",
      lastName: "Eco Bulider",
      phone: "+121 511 4522",
    },
  ];

  return (
    <section className="section-container">
      <div class="contenedor">
        <img src="https://res.cloudinary.com/veronicaduque/image/upload/v1653111006/RealMod/Our_Agents_ixzutp.png" />
        <div class="centrado">
            <span>EXperties is here</span>
            <p>Our Exclusive Agents</p>
            
        </div>
      </div>
      <div className="containerAgents">
        {data.map((item) => (
          <Card sx={{ width: 270, minHeight: 275 }}>
            <img src={item.image} className="image-card-agents" />
            <div className="description-card">
              <p>{item.name}</p>
              <span>{item.lastName}</span>
              <div className="container-contact">
                <img src="https://res.cloudinary.com/veronicaduque/image/upload/v1653259701/RealMod/fi-rr-call-outgoing_uivqzu.png" />
                <p> Call: {item.phone}</p>
              </div>
            </div>
          </Card>
        ))}
      </div>
    </section>
  );
}
