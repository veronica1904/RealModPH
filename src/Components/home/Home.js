import React from 'react'
import SectionFourAgents from './SectionFourAgents';
import SectionOne from './sectionOne';
import SectionTwoHome from './SectionTwoHome';
import SectionRegisterAgent from './SectionRegisterAgent';

export default function Home() {
  return (
    <div>
      <SectionOne />
      <SectionTwoHome/>
      <SectionFourAgents />
      <SectionRegisterAgent />
    </div>
  )
}
