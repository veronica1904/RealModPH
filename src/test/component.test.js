import Home from '../Components/home/Home';
import SectionFourAgents from '../Components/home/SectionFourAgents';
import { getByAltText, render } from '@testing-library/react';

describe('renders Home componets', () => {
 
    it('render componts in home ', () => {
        
        expect(<Home />).toMatchSnapshot();
        
    })
    test('check map text card EXperties is here', ()=> {
        const { getByText } = render(<SectionFourAgents />);
        const loading = getByText(/EXperties is here/i);
        expect(loading).toBeVisible();
    });

});