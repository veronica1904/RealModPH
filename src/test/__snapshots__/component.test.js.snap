// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home componets render componts in home  1`] = `<Home />`;
